Packs/Discord/Integrations/Discord/Discord.py

```diff
@@ -0,0 +1,86 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+import requests
+import json
+import base64
+
+
+def send_message(api_key, channel_id, text):
+    url = f'https://discord.com/api/v9/channels/{channel_id}/messages'
+    headers = {
+        'Authorization': 'Bot ' + api_key,
+        'Content-Type': 'application/json'
+    }
+    payload = {
+        "content": "Your message content here",
+    }
+    response = requests.post(url, headers=headers, data=json.dumps(payload))
```

Comment:
You need to handle the case that the response returned with an error.
There are several options:
- Use `BaseClient`, see an example [here](https://github.com/demisto/content/blob/master/Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.py#L265) - recommended.
- Add `response.raise_for_status()`.
- Add something like this:
`if not response.ok:`
    `raise DemistoException(f'Test Error in API call to Discord: {response.status_code} - {response.reason}.\nFull response: {response.text}')`

Of course also for the second command.